packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [36.0.0]
+
+### Added
+
+- Add optional `topBid` property to the `NftMetadata` type. This property must be of type `TopBid`. ([#4522](https://github.com/MetaMask/core/pull/4522))
+- Add optional `floorAsk` property to the `TokenCollection` type. This property must be of type `FloorAskCollection`. ([#4522](https://github.com/MetaMask/core/pull/4522))
+- Add linea mainnet support to nft detection supported networks ([#4515](https://github.com/MetaMask/core/pull/4515))
+- The `Collection` type is expanded to include the following 'string'-type optional properties: `contractDeployedAt`, `creator`, `ownerCount`, and an optional property `topBid` of the type `TopBid & { sourceDomain?: string; }`. ([#4443](https://github.com/MetaMask/core/pull/4443))
+
+### Changed
+
+- Fetch NFT collections data from the NFT-API `Get Collections` endpoint when calling the `detectNfts` method of `NftDetectionController`, and the `updateNftMetadata` and `watchNft` methods of `NftController`. ([#4443](https://github.com/MetaMask/core/pull/4443))
+- Bump `@metamask/utils` to `^9.0.0` ([#4516](https://github.com/MetaMask/core/pull/4516))
+- Bump `@metamask/rpc-errors` to `^6.3.1` ([#4516](https://github.com/MetaMask/core/pull/4516))
+
+### Fixed
+
+- **BREAKING:** Change `attributes` type in `NftMetadata` to `Attributes[]` ([#4522](https://github.com/MetaMask/core/pull/4522))
```

Comment:
```suggestion
- **BREAKING:** The `attributes` property of the `NftMetadata` type must be of type `Attributes[]` ([#4522](https://github.com/MetaMask/core/pull/4522))
  - This property has been typed as `Attributes` since `@metamask/assets-controllers` `v28.0.0`.
```

Reply:
Was `Attributes` always the wrong type to have here, or are we changing this due to a newly-introduced upstream change?

Reply:
That's good to know, but looks like they're both still in the affected range (v28-v36) since extension is at v34, and mobile is at v30. If they, or any other future consumer in the affected range decide to use `attributes`, they'll need to patch its type to `Attributes[]`. This entry should make them aware of this fact, and whether they should apply this patch, or simply update to `>=36.0.0`.

Reply:
Totally, makes sense 🙏